infra/multichain-dev/main.tf

```diff
@@ -14,6 +14,14 @@ module "gce-container" {
     args  = ["start"]
     port  = "3000"
 
+    volumeMounts = [
```

Comment:
don't we need to do this for more than just `multichain-dev/`?

Reply:
cc @kmaus-near

Reply:
Yes, when it's tested and ready to move into main branch. Eventually I would like to automate our own TF deployments for production, so this is more of a development workflow question we should eventually talk about.